setup.py

```diff
@@ -24,6 +24,7 @@ def readme():
     ],
     extras_require={
         "driver-unipi": ["pyusb", "pymodbus"],
+        "driver-quaddali": ["pyserial"],
```

Comment:
I'd prefer naming the extra requirements after the installed dependencies.

    extras_require={
        'usb': ['pyusb'],
        'serial': ['pyserial'],
        'modbus': ['pymodbus']
    }

Reply:
Consider it done.